dist/dev/index.html

```diff
@@ -1,26 +1,47 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 <head>
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <meta charset="utf-8">
-  <link href="/css/bootstrap.min.css" rel="stylesheet" type="text/css">
-  <link href="/css/main-index.css" rel="stylesheet" type="text/css"/>
-  <link href="/css/main-fonts.css" rel="stylesheet" type="text/css"/>
-  <link href="/css/slick.min.css" rel="stylesheet" type="text/css"/>
-  <link href="/css/slick-theme.min.css" rel="stylesheet" type="text/css"/>
-  <meta property="og:image" content="https://s3.amazonaws.com/stuyspec-images/the_logo.png"/>
-  <meta property="og:title" content="The Stuyvesant Spectator" />
-  <meta property="og:url" content="https://stuyspec.com" />
-  <meta property="og:site_name" content="The Spectator" />
-  <meta property="og:type" content="website" />
+    <meta http-equiv="content-type" content="text/html; charset=UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <meta name="author" content="The Spectator Web Department" />
+    <meta name="description" content="The Pulse of the Student Body" />
+    <meta name="keywords" content="The Spectator, journalism, Stuyvesant High School, student body" />
+
+    <meta property="og:image" content="https://s3.amazonaws.com/stuyspec-images/the_logo.png"/>
+    <meta property="og:title" content="The Stuyvesant Spectator" />
+    <meta property="og:url" content="https://stuyspec.com" />
+    <meta property="og:site_name" content="The Spectator" />
+    <meta property="og:type" content="website" />
+
+    <title>The Stuyvesant Spectator - The Pulse of the Stuyvesant Student Body</title>
+
+    <link href="/css/bootstrap.min.css" rel="stylesheet" type="text/css">
+    <link href="/css/main-index.css" rel="stylesheet" type="text/css"/>
+    <link href="/css/main-fonts.css" rel="stylesheet" type="text/css"/>
+    <link href="/css/slick.min.css" rel="stylesheet" type="text/css"/>
+    <link href="/css/slick-theme.min.css" rel="stylesheet" type="text/css"/>
+
+    <!-- Global Site Tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-49387606-1"></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];

```

Comment:
What is this doing?

Reply:
**Both** of those script tags are for Google Analytics.

Reply:
Link for ref?

Reply:
<https://developers.google.com/analytics/devguides/collection/analyticsjs/#alternative_async_tracking_snippet>

Reply:
Mind putting a link as a comment?

Reply:
I don't think it's necessary. The next script tag has:
```html
<script>
    window.dataLayer = window.dataLayer || [];
    function gtag(){dataLayer.push(arguments);}
    gtag('js', new Date());

    gtag('config', 'UA-49387606-1');
</script>
```
With `gtag` and the `UA-...` code, it should be apparent what the script's purpose is. If you disagree, I'll comment the link in.

Reply:
Eh, alright. I'd prefer a comment but I guess it's related to the other stuff